### PR TITLE
add addons cubefs_v3.2

### PR DESCRIPTION
**Pull request by wangshli**
### Ⅰ. Describe what this PR does
add addons cubefs_v3.2

### Ⅱ. Does this pull request fix one issue?

fixes #XXXX

### Ⅴ. Special notes for reviews

### CODE REVIEW

File: addons/cubefs/v3.2/docker/fluid_config_init.py [hunk 1]
```diff
@@ -0,0 +1,63 @@
+#  Copyright 2022 The Fluid Authors.
```
Comment (cheyang) [inline addons/cubefs/v3.2/docker/fluid_config_init.py:1]: I think the Copyright year should be changed to 2023. thanks.

File: addons/cubefs/v3.2/runtime-profile.yaml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+apiVersion: data.fluid.io/v1alpha1
+kind: ThinRuntimeProfile
+metadata:
+  name: cubefs
```
Comment (cheyang) [inline addons/cubefs/v3.2/runtime-profile.yaml:4]: I suggest naming the profile name as cubefs3.2

File: addons/cubefs/v3.2/docker/fluid_config_init.py (hunk 1 above)
Comment (wangshli) [reply addons/cubefs/v3.2/docker/fluid_config_init.py:1]: Ok

File: addons/cubefs/v3.2/runtime-profile.yaml (hunk 2 above)
Comment (wangshli) [reply addons/cubefs/v3.2/runtime-profile.yaml:4]: ok,  v2.4 runtime-profile also modify

**Review by cheyang — APPROVED** [review]

**Pull request merged**